src/Nethermind/Nethermind.Evm/CodeAnalysis/JumpDestinationAnalyzer.cs

```diff
@@ -35,7 +35,7 @@ public bool ValidateJump(int destination, bool isSubroutine)
             {
                 // Store byte to int, as less expensive operations at word size
                 int codeByte = machineCode[destination];
-                result = isSubroutine ? codeByte == BEGINSUB : codeByte == JUMPDEST;
+                result = codeByte == JUMPDEST;
```

Comment:
Unnecessary comparison now? All jump destinations are now JUMPDEST.

Reply:
Is the comparision to see if is JUMPDEST or other code; just don't need the subroutine part (double checks) any more

Reply:
But the `IsJumpDestination` checks if it is jump destination, no need to check it 2nd time, it will always be true.

Reply:
Ah correct; removed